umh-core/pkg/fsm/benthos/observed_state_snapshot.go

```diff
@@ -38,9 +38,16 @@ func (b *BenthosInstance) CreateObservedStateSnapshot() fsm.ObservedStateSnapsho
 	snapshot := &BenthosObservedStateSnapshot{}
 
 	// Deep copy config
-	deepcopy.Copy(&snapshot.Config, &b.config)
+	err := deepcopy.Copy(&snapshot.Config, &b.config)
```

Comment:
We handle all of the deepcopies different.
This for example just silently handles the error and returns nil.
Might make sense to either return both a value and error or at least report the error.

Reply:
done